spec.html

```diff
@@ -4259,7 +4259,7 @@
         1. Let _keys_ be ? _O_.[[OwnPropertyKeys]]().
         1. If _level_ is `"sealed"`, then
           1. Repeat for each element _k_ of _keys_,
-            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{ [[Configurable]]: *false*}).
+            1. Perform ? DefinePropertyOrThrow(_O_, _k_, PropertyDescriptor{[[Configurable]]: *false*}).
```

Comment:
how many existing examples do we have that have the space versus not? I find it much more readable with the space and it seems, at a glance, that "with the space" is far more common.